src/components/SearchResults.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { List } from 'semantic-ui-react';
+
+export class SearchResults extends Component { 
+  constructor(props) {
+    super(props);
+    this.list = [
+      {
+        _id: 1,
+        name: "yolo",
+        location: "somewhere",
+        distance: 2
+      },
+      {
+        _id: 2,
+        name: "yoyo",
+        location: "nowhere",
+        distance: 3.5
+      }
+    ];
+  } 
+
+  // Takes a list of club and event objects and maps it to a list of xml
+  // elements 
+  ListOfResults = (props) => {
+    const searchResults = props.list.map((result) => 
+      <List.Item key={result._id}>
+        {undefined ? result.image : null}
```

Comment:
In the future, use this comment style:
https://maksimivanov.com/posts/how-to-comment/